pkg/featuregates/featuregates.go

```diff
@@ -0,0 +1,114 @@
+package featuregates
```

Comment:
Not a blocker, but the generic stuff in this repo which crosses components usually goes within support/. We can follow up for that if we want to.